docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM google/dart-runtime-base:2.0.0-dev.8.0
+
+# `apt-mark hold dart` ensures that Dart is not upgraded with the other packages
```

Comment:
If you keep this file one level up we can get the transitive packages in as well. Just use a different name, like `Dockerfile.pub_site_base`. `docker build -f  Dockerfile.pub_site_base` will build it.

Also add the `pubspec.yaml` and the `pub get` in here.